<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInterface</class>
 <widget class="QDialog" name="EditInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="editPhenomena">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="deletePhenomena">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="addPhenomena">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" rowspan="3" colspan="3">
    <widget class="QListView" name="phenomenaListView">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="phenomenaLabel">
     <property name="text">
      <string>Phenomena:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="withoutArrowRadio">
     <property name="text">
      <string>Without Arrow</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="withArrowRadio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>With Arrow</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
